.changeset/lovely-olives-itch.md

```diff
@@ -0,0 +1,7 @@
+---
+"@clerk/clerk-js": patch
+"@clerk/shared": patch
+"@clerk/ui": patch
+---
+
+Moves `fastDeepMerge` utils to `@clerk/shared` package.
```

Comment:
@alexcarpenter 👍 Might want to add an additional changeset for just the `useLocalizations` update

Reply:
NOTE: Removing `"@clerk/ui": patch` from here rather than adding another changeset.